composer/tools/composer_dags.md

```diff
@@ -0,0 +1,83 @@
+# Composer DAGs Pausing/Unpausing script
+
+This document describes usage of composer_dags.py script.
+
+Purpose of the script it to provide a tool to pause or unpause all the DAGs 
+(except airflow_monitoring) of a Composer environment.
+
+## Prerequisites
+
+1.  [Make sure you are authorized](https://cloud.google.com/sdk/gcloud/reference/auth/login)
+    through `gcloud auth login` before invoking the script . The script requires
+    [permissions to access the Composer environment](https://cloud.google.com/composer/docs/how-to/access-control)
+    (roles/composer.environmentAndStorageObjectAdmin role is sufficient) and its
+    GKE cluster (container.*).
+
+1.  Make sure that you can access GKE cluster of Composer environment through
+    kubectl (e.g. if you are not able to successfully execute
+    `kubectl get pods`, the script will fail). "Unable to connect to the server:
+    dial tcp [IP ADDRESS]:443: connect: connection timed out" reported by
+    kubectl indicates that kubectl can not connect to GKE cluster.
+    [Refer to the documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#cloud_shell)
+    about setting up access to private GKE cluster if you have private IP environment.
+
+1.  Supported environment versions: all Composer versions 
```

Comment:
Is it really all Composer all Airflow versions, even if it's a Python 2 environment?

Reply:
Yes. It is not dependent on composer python version, although, The script is only supported by python 3.